Doc/library/textwrap.rst

```diff
@@ -57,15 +67,15 @@ functions should be good enough; otherwise, you should use an instance of
       >>> textwrap.shorten("Hello world", width=10, placeholder="...")
       'Hello...'
 
-   Optional keyword arguments correspond to the instance attributes of
-   :class:`TextWrapper`, documented below.  Note that the whitespace is
-   collapsed before the text is passed to the :class:`TextWrapper` :meth:`fill`
-   function, so changing the value of :attr:`.tabsize`, :attr:`.expand_tabs`,
-   :attr:`.drop_whitespace`, and :attr:`.replace_whitespace` will have no effect.
+   See :class:`TextWrapper` for documentation on all optional keyword
+   arguments. Note that the whitespace is collapsed before the text is passed
+   to the :class:`TextWrapper` :meth:`fill` function, so changing the value of
+   :attr:`.tabsize`, :attr:`.expand_tabs`, :attr:`.drop_whitespace`, and
+   :attr:`.replace_whitespace` will have no effect.  Be careful not to misspell
```

Comment:
The last sentence can be omitted because the same can be said for any func / method in stdlib.